### DISCUSSION TITLE
v24.10.12

### DISCUSSION THREAD

**Discussion by davidrapan** [davidrapan/ha-solarman#209] [Announcements]
## What's Changed
* [fix: Returning new values even when disconnected](https://github.com/davidrapan/ha-solarman/commit/0c79b18a900d7e41c79b2e2f4f8f016389d5fe83)
* [refactor: Value extraction from response sets](https://github.com/davidrapan/ha-solarman/commit/ca3797b4cb90dd7f8c11dd1147a00767346c4fe5)
* Add profile for hybrid inverters from CHINT by @gardiol in https://github.com/davidrapan/ha-solarman/pull/200

## New Contributors
* @gardiol made their first contribution in https://github.com/davidrapan/ha-solarman/pull/200

**Full Changelog**: https://github.com/davidrapan/ha-solarman/compare/v24.10.10...v24.10.12

<hr /><em>This discussion was created from the release <a href='https://github.com/davidrapan/ha-solarman/releases/tag/v24.10.12'>v24.10.12</a>.</em>

**Comment by tgessendorfer** [comment]
All (even almost) my sensors are gone with my Deye 12K since last update!?

![image](https://github.com/user-attachments/assets/4903d379-e9dc-4999-b13e-cb9cf5f92d1c)

**Reply by davidrapan** [reply]
My first guess is that it's caused by name change sometime in the past which was broken so try to remove and re-add device back. 😉

**Comment by tgessendorfer** [comment]
Ahhh... I had the other Solarman HACS integration installed/used as well and there seem to be some overlatch issues so had to clean up and re-install the integrations to get clear and working entites back!
Thanks!
